modules/client-configuration/pages/contact-methods.adoc

```diff
@@ -412,7 +412,7 @@ OSAD allows registered client systems to execute scheduled actions immediately.
 OSAD has several distinct components:
 
 * The [systemitem]``osa-dispatcher`` service runs on the server, and uses database checks  to determine if clients need to be pinged, or if actions need to be executed.
-* The [systemitem]``osad`` service runs on the client. It responds to pings from [systemitem]``osa-dispatcher`` and runs [command]``mgr_check`` to execute actions when directed to do so.
+* The [systemitem]``mgr-osad`` service runs on the client. It responds to pings from [systemitem]``osa-dispatcher`` and runs [command]``mgr_check`` to execute actions when directed to do so.
```

Comment:
`osad`

Reply:
I wonder... why do we have exactly the same content here as at `modules/architecture/pages/traditional-osad.adoc`? 

I think I am reviewing the same things twice :-D

Reply:
Yeah, there does seem to be some duplication here. This is also a massive file, and needs to be broken up.